foca/utils/logging.py

```diff
@@ -0,0 +1,78 @@
+"""Decorator for logging requests and responses."""
```

Comment:
There may be more logging utilities added to this module in the future, so better stay generic here. The decorator is described in its own docstring.